doc/source/user_guide/installation.rst

```diff
@@ -35,6 +35,9 @@ Now activate the LumiSpy environment and install the package using pip:
 
 Installation is completed! To start using it, check the next section.
 
+.. Note::
+
+    If the installation fails using ``pip``, try installing using ``conda install lumispy -c conda-forge``.
```

Comment:
I can see two situations where it can happen:
- some lumipsy dependencies can't be installed, for example, compiler missing.
- seems to install OK, but pip broke the python installation because of ABI incompatibility, for example, it is not possible to import numpy, scipy or numba anymore.

Would it be worth being more specific and explain a bit more? As it currently, I don't understand what is the logic.